Gemfile

```diff
@@ -66,5 +66,8 @@ end
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem 'capybara'
+  gem 'database_cleaner'
   gem 'selenium-webdriver'
 end
+
+gem 'bullet', group: :development
```

Comment:
Nice job updating your readme file as requested by the previous reviewer😉. You still haven't implemented the changes the previous reviewer pointed out😉. Just as requested in the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md), you are meant to take away the `bullet gem` and solve the N+1 concerns without it😉. just as the previous reviewer has suggested, for example, you can make use of  `@posts = @user.posts.includes(:comments)` instead of `@posts = @user.posts`😉. Please do well to work with this method and take a look at[ this article](https://dev.to/junko911/rails-n-1-queries-and-eager-loading-10eh) for clarity👍🏼.


<img width="743" alt="image" src="https://github.com/Errytagedesign/blogApp/assets/64643995/54431065-54e5-4a86-a1fc-6e7675218064">